okapi-common/src/main/java/org/folio/okapi/common/OkapiToken.java

```diff
@@ -15,25 +14,21 @@
 public class OkapiToken {
   private String token;
 
-  public OkapiToken() {
-    this.token = null;
-  }
-
-  public OkapiToken(RoutingContext ctx) {
-    this.token = ctx.request().getHeader(XOkapiHeaders.TOKEN);
-  }
-
-  public void setToken(String token) {
+  public OkapiToken(String token) {
     this.token = token;
   }
 
   private JsonObject getPayload() {
```

Comment:
This returns the payload without validating the JWT using the signature. This may be a security issue if used in other contexts.
Can the method been renamed to `getPayloadWithoutValidation` and javadoc added that there is no JWT signature validation?

Reply:
ok.. getPayload renamed.. Internal so nothing is affected.